### DISCUSSION TITLE
How can '|' be used to separate commands in cmdline-lines?

### DISCUSSION THREAD

**Discussion by kz-worn** [JetBrains/ideavim#716] [Q&A]
I want to work multiple substitutions at once.
Example:
:%s/table/xxx/g|%s/user/yyy/g

**Comment by lippfi** [comment]
I've created a ticket for this issue https://youtrack.jetbrains.com/issue/VIM-3120/Allow-pipes-to-separate-multiple-subsitute-commands-in-one-line
Thank you for this report!

**Reply by kz-worn** [reply]
Thanks！
